packages/react-test-renderer/src/ReactShallowRenderer.js

```diff
@@ -28,6 +30,16 @@ class ReactShallowRenderer {
     this._rendering = false;
     this._forcedUpdate = false;
     this._updater = new Updater(this);
+
+    if (__DEV__) {
+      if (warnAboutDeprecatedLifecycles) {
```

Comment:
Why is it important for these to be isolated?

Reply:
It's not, but it does result in slightly cleaner dev-bundle:
```js
if (warnAboutDeprecatedLifecycles) {
```
Rather than
```js
if (true && warnAboutDeprecatedLifecycles) {
```
We do have them in isolation for at least one other feature flag. That being said, I don't feel very strongly about this. It just seemed slightly nicer.

Reply:
Sorry, I meant why is it important to make them fields instead of top-level like we usually do.

Reply:
Oh! I misunderstood. 😄 

No strong reason here. I'll move them off the instance.